### ISSUE TITLE
Improve instructions to install TestNG in Eclipse IDE

### ISSUE THREAD

**Issue by darecoder1999** [OpenRefine/OpenRefine#2463]
**Describe the bug**
https://github.com/OpenRefine/OpenRefine/wiki/Building-OpenRefine-From-Source#testing-in-eclipse Link mention here for installing TestNG gives error 410 as it is no longer available.

**Expected behavior**

The mentioned link should be replaced with the correct one.

**Screenshots**

![410error](https://user-images.githubusercontent.com/42637982/77342812-c781a980-6d56-11ea-8458-df77bf629e1f.png)

**Comment by darecoder1999** [comment]
Modifications done. 
Please review, if it's correct or not.

**Comment by antoine2711** [comment]
@darecoder1999: for modification to the wiki, there is not really a process approval. But, I'd say @ostephens is pretty much leading that aspect, though is mostly on the new documentation. There is also Thad who look a lot at the wiki, and he will probably stumble on your modification at one point.

I've check you modification, and they where good. I saw that for the link « Installing New Software », you pointed to the « Upgrade » page of that tool. I've changed it for the menu. But other than that, it was excellent. 

Nice job. Regards, Antoine.
